cluster_tools/downscaling/downscaling_workflow.py

```diff
@@ -128,25 +129,26 @@ def _have_scale(self, scale):
         with vu.file_reader(self.input_path) as f:
             return key in f
 
-    def _copy_scale_zero(self, out_path, out_key, dep):
+    def _copy_scale_zero(self, out_path, out_key, dep,int_to_uint):
```

Comment:
```suggestion
    def _copy_scale_zero(self, out_path, out_key, dep, int_to_uint):
```

Reply:
PEP8, I suggest you use a linter before commiting, e.g. https://pypi.org/project/pyflakes/
(But with linewidth=120)